### PR TITLE
new volume path on docker-compose files

### PR DESCRIPTION
**Pull request by sistemas-baobab**
New volume path to mount example cornflow-dags folder into docker-airflow service

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -37,9 +37,9 @@ services:
       dockerfile: Dockerfile
     restart: always
     volumes:
-      - ./airflow_config/dags:/usr/local/airflow/dags
-      - ./airflow_config/requirements.txt:/requirements.txt
-      - ./airflow_config/plugins:/usr/local/airflow/plugins
+      - ./cornflow-dags/DAG:/usr/local/airflow/dags
+      - ./cornflow-dags/requirements.txt:/requirements.txt
+      - ./cornflow-server/airflow_config/plugins:/usr/local/airflow/plugins
```
Comment (ggsdc) [inline docker-compose.yml:42]: This last line, should not need to be changed as the context of the docker compose file is `./cornflow-server/airflow_config`, no?

Comment (ggsdc) [reply docker-compose.yml:42]: This same thing could be said about the rest of the docker compose files

**Review by ggsdc — COMMENTED** [review]
We should talk about this on tuesday.

File: docker-compose.yml (hunk 1 above)
Comment (ampuerin) [reply docker-compose.yml:42]: Now, the context of docker-compose files is in root repo folder. We have to load plugins from ./cornflow-server/airflow_config/plugins.
For DAG and requirements, now been moved to new DAG folder in the repo we load them from ./cornflow-dags/DAG and ./cornflow-dags/requirements.txt.

**Review by ggsdc — APPROVED** [review]

**Pull request merged**